sci-mathematics/cvc4/files/cvc4-1.8-bash-5.2-fix.patch

```diff
@@ -0,0 +1,36 @@
+diff --git a/src/expr/mkexpr b/src/expr/mkexpr
```

Comment:
Please add links to:
1. any relevant Gentoo bugs
2. the patch source
at the top of the patch file.

Reply:
Done, I noticed that not all patches include a link to the Gentoo bug though. Can you kindly explain when it's needed and when it's not?

Reply:
It's kind of a newer thing that we do. See https://devmanual.gentoo.org/ebuild-writing/misc-files/patches/index.html#clean-patch-howto.

It overall makes it easier to understand why a patch is there & if we still need it.

Reply:
ok, thank you